[2021-08-28 10:53:38,038] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: test.training_model_A 2021-07-21T00:00:00+00:00 [queued]>
[2021-08-28 10:53:38,041] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: test.training_model_A 2021-07-21T00:00:00+00:00 [queued]>
[2021-08-28 10:53:38,041] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-28 10:53:38,041] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-28 10:53:38,041] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-28 10:53:38,054] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_A> on 2021-07-21T00:00:00+00:00
[2021-08-28 10:53:38,055] {standard_task_runner.py:52} INFO - Started process 1315 to run task
[2021-08-28 10:53:38,056] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'training_model_A', '2021-07-21T00:00:00+00:00', '--job-id', '67', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/example.py', '--cfg-path', '/tmp/tmpglacdk3w', '--error-file', '/tmp/tmp72i7qutd']
[2021-08-28 10:53:38,056] {standard_task_runner.py:77} INFO - Job 67: Subtask training_model_A
[2021-08-28 10:53:38,076] {logging_mixin.py:104} INFO - Running <TaskInstance: test.training_model_A 2021-07-21T00:00:00+00:00 [running]> on host DESKTOP-4B4VMOV.localdomain
[2021-08-28 10:53:38,107] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-07-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-21T00:00:00+00:00
[2021-08-28 10:53:38,108] {python.py:151} INFO - Done. Returned value was: 10
[2021-08-28 10:53:38,125] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=test, task_id=training_model_A, execution_date=20210721T000000, start_date=20210828T085338, end_date=20210828T085338
[2021-08-28 10:53:38,144] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-28 10:53:38,147] {local_task_job.py:149} INFO - Task exited with return code 0
